### ISSUE TITLE
Should Role Name be in Quotes in Set-Role Help Documentation?

### ISSUE THREAD

**Issue by CBPSC** [dnndev/Dnn.AdminExperience.Extensions#39]
![set-quotes](https://cloud.githubusercontent.com/assets/4571863/22981068/4ee017f6-f369-11e6-844f-f7da68a30f4b.png)

**Comment by dnndev** [comment]
No. It can be, but it's not required. If the role name was something like Content Editor then it would need to be, because it contains a space.

**Comment by CBPSC** [comment]
Gotcha... makes sense. Thanks!

**Comment by david-poindexter** [comment]
In my opinion, this should be clarified in the --name Description.  Something along the lines of "Use quotes when the role name contains spaces".

**Comment by dnndev** [comment]
This would need to be added to every option on every command that can have a string value.  It is described in the Syntax topic `help syntax`.

**Comment by dnndev** [comment]
I'm open to options

**Comment by david-poindexter** [comment]
Good point.  Seems like some tedious work, but most likely worthwhile for newbies.  If you want, we can just create an issue for each command to handle these changes.  I would prefer this over one single issue for all of them.  I can work on this in spurts better than taking a long time to do all of them.  Thoughts?

**Comment by dnndev** [comment]
I worry about help bloat. At a certain point, people have to understand the rules by which the program operates. Some possibilities may be that we make the docs more consistent (if they're not already)... that strings are never quoted unless they need to be. Or maybe we add a link to each topic that links to the syntax section describing string usage. Or... ?

**Comment by david-poindexter** [comment]
@dnndev totally agreed.  I like the idea of a link being added to each topic.
